### ISSUE TITLE
Add --dev argument and behave similar to composer by ignoring development dependencies

### ISSUE THREAD

**Issue by clue** [clue/graph-composer#2]
The current behavior is to also visualize development dependencies. This should be changed to ignore them by default and add a new `--dev` argument flag which enables showing development dependencies.

**Comment by clue** [comment]
> any news?

Yeah, I've pushed an upstream PR (https://github.com/schmittjoh/composer-deps-analyzer/pull/9), but haven't received any feedback so far.

Meanwhile, there's also a pending PR (#19) which implements this in our code base instead of pushing this upstream. I suppose any input is welcome :) Perhaps coordinate efforts and pick up either / decide on either PR? :+1:

**Comment by GaryJones** [comment]
Not much to add, except a strong +1 for being able to ignore `require-dev` dependencies of my package and those of the `require` packages my package depends on.
